### ISSUE TITLE
Change text layout in "Merkliste anlegen"

### ISSUE THREAD

**Issue by owlRike** [lokalkauf/lokalkauf-frontend#224]

**Comment by christianaschoff** [comment]
That is not an issue of merkliste, but of the whole application, because the chosen font and size does not match the given space. 
can you rephrase that issue to make it a global change throughout the application? Otherwise it would be a pain to just fix it in one specific input-field?
Thank you :)

**Comment by owlRike** [comment]
better? :-)

**Comment by christianaschoff** [comment]
changed line height for all text-inputs. Need to have a good look at it on all browsers

**Comment by Dipanjana1990** [comment]
Not sure if this question is related to this issue.
![image](https://user-images.githubusercontent.com/68644410/99399750-5e9f0100-28e6-11eb-8c39-bccbb449ecda.png)

Should this be in the same line? Tested in Android

**Comment by christianaschoff** [comment]
Hello @Dipanjana1990 please open another ticket for that issue :)
I guess that will be a case for a general design overhaul
